src/components/spreadsheet/equipment-tabs.tsx

```diff
@@ -41,6 +42,12 @@ export const EquipmentTabs: FunctionComponent<EquipmentTabsProps> = ({ tabIndex,
                 >
                     {tablesNames.map((table) => (
                         <Tab
+                            sx={{
```

Comment:
Instead of handling this with css, you can just filter the tables names ?

`
  const visibleTables = tablesNames.filter(
    (table) => table !== TABLES_DEFINITIONS.BUSBAR_SECTION.name
  );
`

Reply:
With this logic, we can't create an empty spreadsheet named 'BusBarSections.' We will receive the error message 'Spreadsheet name already exists,' even though it's not visible. (To discuss with the Product Owner)

Reply:
Busbar sections table is now always visible